<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexWidget</class>
 <widget class="QWidget" name="IndexWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelSite">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>50</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string>当前位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLocation">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>138</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangeLocation">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>71</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string>[切换地址]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSearch">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>15</y>
     <width>290</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
